samples/java-valueentity-shopping-cart/src/it/java/com/example/valueentity/shoppingcart/domain/ShoppingCartIntegrationTest.java

```diff
@@ -25,18 +25,25 @@
 import java.util.List;
 
 import static org.junit.Assert.assertEquals;
+import static java.util.concurrent.TimeUnit.*;
 
+// Example of an integration test calling our service via the Akka Serverless proxy
+// Run all test classes ending with "IntegrationTest" using `mvn verify -Pfailsafe`
```

Comment:
```suggestion
// Run all test classes ending with "IntegrationTest" using `mvn verify -Pit`
```
I guess the codegen needs to be fixed...

Reply:
https://github.com/lightbend/akkaserverless-codegen/pull/142